test/sslapitest.c

```diff
@@ -4555,6 +4557,179 @@ static int test_ssl_get_shared_ciphers(int tst)
     return testresult;
 }
 
+static const char *appdata = "Hello World";
+static int gen_tick_called, dec_tick_called, tick_key_cb_called;
+static int tick_key_renew = 0, tick_dec_renew = 0;
+
+static int gen_tick_cb(SSL *s, void *arg)
+{
+    gen_tick_called = 1;
+
+    return SSL_SESSION_set1_ticket_appdata(SSL_get_session(s), appdata,
+                                           strlen(appdata));
+}
+
+static int dec_tick_cb(SSL *s, SSL_SESSION *ss, const unsigned char *keyname,
```

Comment:
With the latest changes, the return value of `dec_tick_cb` is once again `SSL_TICKET_RETURN`.

Reply:
Fixed.